package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-wfm-user",
-  "version": "0.0.25",
+  "version": "0.0.26",
```

Comment:
I'd say leave version bumping for when the module is being published as part of the release process.